src/main/scala/com/itechart/ny_accidents/service/PopulationService.scala

```diff
@@ -19,6 +19,6 @@ object PopulationService {
   }
 
   def calculateDensity(population: Population): Double = {
-    population.population.toDouble / (population.area * NUMBER_NORMALIZATION_FACTOR)
+    (((population.population.toDouble / (population.area * NUMBER_NORMALIZATION_FACTOR))*100).toInt) /100.0
```

Comment:
You multiple on 100 the device on 100. I guess we can get rid of it. If you try round number in this way, then there is appropriate methods for that: round/ceil/floor etc. Please, correct me if I am wrong.